mewarpx/mewarpx/diags_store/diag_base.py

```diff
@@ -110,7 +111,7 @@ def _comp_calcinterval(self, it):
             next_diagnum = ((diagnum // diagnum_divisor) + 1)*diagnum_divisor
             next_itnum = next_diagnum * self.diag_steps + self.diag_step_offset
 
-            print(
+            logger.info(
```

Comment:
While we're changing these print statements, should we change these string.format functions to just fstrings?